test/e2e/rc.go

```diff
@@ -26,17 +26,41 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/client"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
-	"github.com/golang/glog"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
+var _ = Describe("ReplicationController", func() {
+	var (
```

Comment:
I think @roberthbailey means:

```go
var _ = Describe("ReplicationController", func() {
        var c *client.Client
        BeforeEach(func() {
                ...
```

But I don't know if I agree that that would be better, because as you add a second variable you'd have to expand the block again, so personally I'm happy with the `var ( ... )` block as it is right now.